pwa-update-available/PwaUpdateAvailable.js

```diff
@@ -8,7 +8,11 @@ export class PwaUpdateAvailable extends HTMLElement {
 
   async connectedCallback() {
     this.setAttribute('hidden', '');
-    this.addEventListener('click', this._postMessage.bind(this));
+    this.auto = this.hasAttribute('auto');
+    this.duration = this.getAttribute('duration') || 0;
```

Comment:
`this.getAttribute('duration')` returns a string, right?

Reply:
yeah it comes from attribute so... I should apply parseInt ?